test/specs/cli.js

```diff
@@ -317,6 +331,12 @@ describe('cli', () => {
         .equal('\nsomething\nsomething\n');
     });
 
+    it('does not work with empty regex strings', () => {
+      (() => {
+        cli('sed', 's//foo/g', testFileName1);
+      }).should.throw(Error);
```

Comment:
I would check for the exact error message here, i.e. `.should.throw('Bad sed pattern (empty regex)')`.

Reply:
done